plugins/coral-plugin-viewing-options/client/components/ViewingOptions.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import cn from 'classnames';
+import {Icon} from 'coral-ui';
+import styles from './ViewingOptions.css';
+import Slot from 'coral-framework/components/Slot';
```

Comment:
We need to provide this through the `coral-plugin-api` in a folder called `alpha` or `beta` if we are more confident that it doesn't change.